docs/eventing/brokers/broker-admin-config-options.md

```diff
@@ -12,8 +12,6 @@ kind: ConfigMap
 metadata:
   name: config-br-defaults
   namespace: knative-eventing
-  labels:
```

Comment:
This whole page should move to **Eventing > Administrator topics > Eventing configuration**

Reply:
:+1: Done